.gitignore

```diff
@@ -15,4 +15,7 @@ project/plugins/project/
 .history
 .cache
 .lib/
+.idea/
```

Comment:
Can you use global `.gitignore` rather then committing idea specific settings, see https://stackoverflow.com/a/7335487/1519631